backend/cmd/worker/sequencing_experiment_validation.go

```diff
@@ -0,0 +1,433 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"regexp"
+	"time"
+
+	"github.com/golang/glog"
+	"github.com/radiant-network/radiant-api/internal/repository"
+	"github.com/radiant-network/radiant-api/internal/types"
+	"gorm.io/gorm"
+)
+
+const (
+	AliquotRegExp  = "^[A-Za-z0-9\\-_.]+$" // Alphanumeric characters, hyphens, underscores and periods
+	RunAliasRegExp = "^[A-Za-z0-9\\-_.]+$"
+)
+
+var (
+	AliquotRegExpCompiled    = regexp.MustCompile(AliquotRegExp)
+	CaptureKitRegExpCompiled = AliquotRegExpCompiled
+	RunAliasRegExpCompiled   = AliquotRegExpCompiled
```

Comment:
Is RunAliasRegExp really useful since you re-use AliquotRegExpCompiled so AliquotRegExp ?